module/move/wca/tests/inc/commands_aggregator/help.rs

```diff
@@ -59,3 +59,76 @@ wca = {{path = "{}"}}"#,
     result
   );
 }
+
+#[ test ]
```

Comment:
I think It would be better if we refactor these tests to be able to see what the input is

Reply:
That's not what I meant. Now we have to support this comment as well, and that's bad. I mean, we can modify the test to create the crate at the arrangement stage of a test